internal/expr/expr_test.go

```diff
@@ -59,8 +63,8 @@ var tests = []struct {
 	typeSamples:    []any{Person{}},
 	expectedSQL:    "SELECT p.address_id AS _sqlair_0, p.id AS _sqlair_1, p.name AS _sqlair_2",
 }, {
-	summary:        "spaces and tabs",
-	query:          "SELECT p.* 	AS 		   &Person.*",
+	summary: "spaces and tabs",
+	query: "SELECT p.* 	AS 		   &Person.*",
```

Comment:
For some reason `gofmt` is doing this, i.e. removing the tab, [seems to be this issue](https://github.com/golang/go/issues/51910). This is a new issue for us because we only introduced the named fields a few PRs ago.

If this behaves the same on your PC then I think it'd be best just to leave it rather than fight gofmt. The issue is closed but its still broken for me with the latest version of go installed. (Check on your PC though, I'm assuming that nvim is using the snap gofmt but I'm not actually certain.)

Reply:
In my case running `go fmt` actually changed the line, that is why I commented in the first place :P

Reply:
Ah yea, will see if I can get that fixed in nvim